### ISSUE TITLE
Examples of client registration using different grant

### ISSUE THREAD

**Issue by lo-simon** [AMWA-TV/nmos-authorization#103]
Would it be useful to replace register-client-post-request.json with the following 2 examples, one for using client_credentials grant and one for using authorization_code grant?

register-client-credentials-grant-client-post-request.json
```
{
 "client_name": "My Example Client",
 "grant_types": ["client_credentials"],
 "jwks_uri": "https://client.example.org/my_public_keys.jwks",
 "response_types": ["none"],
 "scope": "registration",
 "token_endpoint_auth_method": "private_key_jwt"
} 

```
register-authorization-code-grant-client-post-request.json
```
{
 "client_name": "My Example Client",
 "grant_types": ["authorization_code", "refresh_token"],
 "redirect_uris": [
   "https://client.example.org/callback",
   "https://client.example.org/callback2"
 ],
 "response_types": ["code"],
 "scope": "query connection",
 "token_endpoint_auth_method": "client_secret_basic"
}
```

**Comment by peterbrightwell** [comment]
Discussed on call -- this would only mean a change to an example. We agree so @simonlo-sony please do PR

**Comment by peterbrightwell** [comment]
Closed by #104
